components/automate-ui/src/app/page-components/projects-filter-dropdown/projects-filter-dropdown.component.ts

```diff
@@ -83,11 +83,8 @@ export class ProjectsFilterDropdownComponent {
   }
 
   handleClearSelection() {
```

Comment:
This is the fix!

Reply:
I noticed the Apply Changes button stays disabled after clicking Clear Selection. Shouldn't it be enabled, in case you want to turn off all filters?

Reply:
Good call!